tasks/timemachine.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Time Machine Directories
+  file:
+    path: "{{ timemachine_data_directory }}"
+    state: directory
+
+- name: Time Machine Docker Container
+  docker_container:
+    name: timemachine
+    image: mbentley/timemachine
```

Comment:
Please can you specify `:latest` for the Docker image

Reply:
Done.